infrastructure/resource_group_level/modules/webapps/dtfs-central-api.bicep

```diff
@@ -0,0 +1,105 @@
+param location string
+param environment string
+param containerRegistryName string
+param appServicePlanEgressSubnetId string
+param appServicePlanId string
+param privateEndpointsSubnetId string
+param cosmosDbAccountName string
+param cosmosDbDatabaseName string
+param logAnalyticsWorkspaceId string
+param externalApiHostname string
+param azureWebsitesDnsZoneId string
+param nodeDeveloperMode bool
+
+param resourceNameFragment string = 'dtfs-central-api'
+
+// These values are taken from GitHub secrets injected in the GHA Action
+@secure()
+param secureSettings object = {
+
+}
+
+// These values are taken from an export of Configuration on Dev (& validating with staging).
+@secure()
+param additionalSecureSettings object = {
+  DOCKER_REGISTRY_SERVER_PASSWORD: 'test-value'
+}
+
+var dockerImageName = '${containerRegistryName}.azurecr.io/${resourceNameFragment}:${environment}'
+var dockerRegistryServerUsername = 'tfs${environment}'
+
+// https://learn.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16
+var azureDnsServerIp = '168.63.129.16'
+
+// These values are hardcoded in the CLI scripts
+var settings = {
+  WEBSITE_DNS_SERVER: azureDnsServerIp
+  WEBSITE_VNET_ROUTE_ALL: '1'
+  PORT: '5000'
+  WEBSITES_PORT: '5000'
+}
```

Comment:
These values are in every webapp. I'm somewhat minded to move them into `webapp.bicep` but currently it doesn't do any processing of inputs, so it's quite nice to keep this outside. Happy to be convinced otherwise!